.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+jobs:
+    build:
+        docker:
+            - image: circleci/ruby:latest
+        steps:
+            - checkout
+            - run:
+                name: Which ruby?
+                command: ruby -v
+            - run:
+                name: Which bundler?
+                command: bundle -v
```

Comment:
Open question: wondering if we want to keep that (assuming it was for debugging the migration)? I guess it's harmless…

Reply:
for bundler I am not sur this version check is necessary but for ruby, because of latest image set on `config.yml`, I wanted to show on which version CI execute build and tests. So do we keep the bundler version ? I have no arguments to keep it.